tests/lib/processor.js

```diff
@@ -330,6 +330,18 @@ describe("processor", () => {
             assert.strictEqual(blocks[0].filename, "0.js");
         });
 
+        it("should the language to its file extension with leading whitespace and trailing characters", () => {
```

Comment:
Maybe a missing word?
```suggestion
        it("should translate the language to its file extension with leading whitespace and trailing characters", () => {
```